ios/brave-ios/Sources/BraveVPN/Components/Settings/BraveVPNSettingsViewController.swift

```diff
@@ -307,8 +309,9 @@ public class BraveVPNSettingsViewController: TableViewController {
           },
           image: BraveVPN.serverLocation.isoCode?.regionFlagImage
             ?? UIImage(braveSystemNamed: "leo.globe"),
-          accessory: .disclosureIndicator,
-          cellClass: MultilineSubtitleCell.self,
+          accessory: .view(BraveVPNSmartProxyCellView(settingsController: self)),
+          cellClass: BraveVPNSmartProxyCell.self,
+          context: ["isSmartProxyAvailable": smartProxyAvailable],
```

Comment:
`"isSmartProxyAvailable"` should be a defined constant, as I think we try to read this from key `"smartRoutingProxyState"`?